src/pages/Defi/hooks/useEarnBalances.tsx

```diff
@@ -26,8 +26,7 @@ export type SerializableOpportunity = MergedEarnVault
 
 export function useEarnBalances(): UseEarnBalancesReturn {
   const { isLoading: isFoxyBalancesLoading, data: foxyBalancesData } = useFoxyBalances()
-  const { vaults, totalBalance: vaultsTotalBalance, loading: vaultsLoading } = useVaultBalances()
-  const vaultArray: SerializableOpportunity[] = useMemo(() => Object.values(vaults), [vaults])
+  const { totalBalance: vaultsTotalBalance, loading: vaultsLoading } = useVaultBalances()
```

Comment:
This should also be deleted, `useVaultsBalances` was only responsible for Yearn vaults, which we've now removed - we're already getting the opportunities balances in this module, so this would most likely result in double balances:

https://github.com/shapeshift/web/blob/e8be8ac4e7746c3341818fca12164ce6da4536dd/src/pages/Defi/hooks/useEarnBalances.tsx#L50-L52